ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/proxy/provision/cucushift-installer-rehearse-ibmcloud-ipi-proxy-provision-chain.yaml

```diff
@@ -12,8 +13,11 @@ chain:
   - ref: enable-qe-catalogsource
   - chain: cucushift-installer-check
   - ref: cucushift-installer-check-ibmcloud
+  - ref: cucushift-installer-check-ibmcloud-resourcegroup
   env:
   - name: RESTRICTED_NETWORK
     default:  "yes"
+  - name: CREATE_CLUSTER_RESOURCE_GROUP
+    default:  "yes" 
```

Comment:
Per `ibmcloud-provision-resourcegroup`, its default value is "no", so here it should be "no" to keep consistent.